.github/workflows/fcos-build.yml

```diff
@@ -0,0 +1,34 @@
+name: FCOS Build from ${{ env.COPR_OWNER }}/${{ env.COPR_PROJECT }}
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  COPR_OWNER: "rhcontainerbot"
+  COPR_PROJECT: "podman-next"
+
+jobs:
+  wait-for-task:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up wait-for-copr
+      run: |
+        pip3 install git+https://github.com/packit/wait-for-copr.git@main
+
+    - name: Wait for ${{ env.COPR_OWNER }}/${{ env.COPR_PROJECT }} build
+      run: |
+        # wait-for-copr attempts 180 tries with 10s between each try by default
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman `git rev-parse --short HEAD`
```

Comment:
If it times out or the tool breaks, presumably you get an `exit(1)` and the workflow fails?  Reason I ask is, nobody/nothing is currently monitoring GHA workflows for failures.  It could fail for months and nobody (except maybe you) would notice (or know what to do with it).  Complicating podman-monitor notification is the frequency this job runs, [an e-mail on failure setup](https://github.com/containers/podman/blob/0fb6708e5299d24f8285ef7ca732e9c86c77b983/.github/workflows/check_cirrus_cron.yml#L81-L92) would spam the list.

* Is the per-push granularity important to the outcome of this job?  i.e. Does someone need to know down to the merge-commit level, when a failure happened?
* Could the job instead run on a cron schedule?  This would greatly improve the option for e-mailing podman-monitor on failure.